app/Http/Controllers/Api/Instructor/CourseController.php

```diff
@@ -80,10 +80,13 @@ public function index(IndexRequest $request)
      */
     public function show(ShowRequest $request): CourseShowResource
     {
-        $instructorId = Auth::guard('instructor')->user()->id;
+        $instructorId = Auth::guard('instructor')->id();
 
         $course = Course::with(['chapters.lessons'])->findOrFail($request->course_id);
 
+        // 認可チェック
+        $this->authorize('show', $course);
```

Comment:
非常に良くできていると思います！
お手数をおかけしますが、メソッド名を`show`から`view`に変更をお願いします🙇‍♂️
こちらは他の実装と一貫性を保つためでした。

Reply:
説明不足ですみません...！
ポリシー側の関数名を変更という内容でしたので、次のように修正をお願いします！
その場合、ポリシーファイル側の関数定義も変更する必要があるので、合わせて修正をお願いします。
```suggestion
        $this->authorize('view', $course);
```